onset_maproom/maproom.py

```diff
@@ -292,6 +292,16 @@ def write_map_title(search_start_day, search_start_month, map_choice, probExcThr
     return mytitle
 
 
+@APP.callback(
+    Output("map_description", "children"),
+    Input("map_choice", "value"),
+)
+def write_map_description(map_choice):
+    for i, text in enumerate(CONFIG["map_text"]):
```

Comment:
You never use `i`, so you don't need `enumerate`.
```
for text in CONFIG["map_text"]:
    if text["menu_value"] == map_choice:
        return text["description"]
```
Furthermore, if you rewrite the config file as
```
map_text:
    monit:
        menu_label: Has germinating rain occured?
        description: The map shows the result of the germinating rain date search
                (i.e. near-real time thus without checking of follow-up dry spells)
                from the most recent Early Start to now
                (or the last day with available rainfall data).
                Dates are expressed in days since an Early Start date.
```
Then you can eliminate the loop entirely and just write
```
 return CONFIG["map_text"][map_choice]["description"]
```